locale/en/docs/guides/simple-profiling.md

```diff
@@ -276,5 +276,8 @@ Hopefully, through the performance investigation of this (admittedly contrived)
 example, you've seen how the V8 tick processor can help you gain a better
 understanding of the performance of your Node.js applications.
 
+You may also find [how to create a flame graph][diagnostics flamegraph] helpful.
+
 [profiler inside V8]: https://v8.dev/docs/profile
 [benefits of asynchronous programming]: https://nodesource.com/blog/why-asynchronous
+[diagnostics flamegraph]: https://nodejs.org/en/docs/guides/diagnostics-flamegraph/
```

Comment:
Seems OK to me. Might be better as a relative link so that when this is copied/translated, the link doesn't need to be changed?

```markdown
[diagnostics flamegraph]: ./diagnostics-flamegraph/
```